src/main/java/org/terasology/launcher/LauncherInitTask.java

```diff
@@ -87,7 +87,7 @@ protected LauncherConfiguration call() {
             // validate the settings
             LauncherSettingsValidator.validate(launcherSettings);
 
-            final boolean serverAvailable = DownloadUtils.isJenkinsAvailable();
+            final boolean serverAvailable = new JenkinsRepository().isAvailable();
```

Comment:
Using JenkinsRepository directly here is kludgy, is there a preferred way for this code to ping the repository?

Reply:
Is `serverAvailable` used somewhere else in this method except for the part where we check for a launcher update? Because that part only talks to Github but not to our Jenkins anymore...

Game release packages are drawn from Jenkins though... 🤔 Ultimately, the code should be refactored that the init task is unaware where network traffic happens, and whether a server is unavailable, it would just populate the game versions from whatever response we get (if that's none, there are simply no games...)

Reply:
At the end, just because the server "is available" right here does not mean that subsequent calls will succeed, so we'll need to safeguard there anyways 🤷